Index: run.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from app import create_app\r\n\r\napp = create_app()\r\n\r\nif __name__ == '__main__':\r\n    app.run(\"0.0.0.0\", debug=True)\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/run.py b/run.py
--- a/run.py	(revision b4ca15443c7f82d2a311f10c8f9a3c11f98dfb41)
+++ b/run.py	(date 1721668636546)
@@ -3,5 +3,5 @@
 app = create_app()
 
 if __name__ == '__main__':
-    app.run("0.0.0.0", debug=True)
+    app.run(host="0.0.0.0", debug=True)
 
